<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(41, 41, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,2,2,2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/image/image/logo.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 255, 0);

border:2px groove gray;
border-radius:10px;
padding:2px 4px</string>
          </property>
          <property name="text">
           <string>全网搜索</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="personalCenter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 255, 0);

border:2px groove gray;
border-radius:10px;
padding:2px 4px</string>
          </property>
          <property name="text">
           <string>个人中心</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 255, 0);

border:2px groove gray;
border-radius:10px;
padding:2px 4px</string>
          </property>
          <property name="text">
           <string>历史记录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
        <property name="spacing">
         <number>14</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1031</width>
             <height>541</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QListWidget" name="listWidget">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;宋体&quot;;
</string>
                </property>
                <item>
                 <property name="text">
                  <string>首页</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>热门短视频</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>电视剧</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>电影</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>综艺</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>少儿</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>动漫</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>纪录片</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>体育</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>游戏</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>排行榜</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>音乐</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>全部频道</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="browerser">
                <property name="text">
                 <string>浏览器</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_3">
               <widget class="QWidget" name="widget_2" native="true">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>10</y>
                  <width>851</width>
                  <height>521</height>
                 </rect>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <widget class="QWidget" name="horizontalLayoutWidget_2">
                 <property name="geometry">
                  <rect>
                   <x>-10</x>
                   <y>0</y>
                   <width>791</width>
                   <height>261</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="7,3">
                  <property name="spacing">
                   <number>12</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="firstLogo">
                    <property name="styleSheet">
                     <string notr="true">background-color:rgb(41, 41, 49);</string>
                    </property>
                    <property name="text">
                     <string>首页推荐</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,1,1,1,1">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetMinAndMaxSize</enum>
                    </property>
                    <item>
                     <widget class="QPushButton" name="pushButton1">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="contextMenuPolicy">
                       <enum>Qt::DefaultContextMenu</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 56);
</string>
                      </property>
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton2">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 56);
</string>
                      </property>
                      <property name="text">
                       <string>2</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton3">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 56);</string>
                      </property>
                      <property name="text">
                       <string>3</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton4">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 56);</string>
                      </property>
                      <property name="text">
                       <string>4</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton5">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>5</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton6">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>6</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton7">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>7</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="horizontalLayoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>-1</x>
                   <y>290</y>
                   <width>781</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(41, 41, 49);</string>
                    </property>
                    <property name="text">
                     <string>热门推荐</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="upButton">
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(41, 41, 49);</string>
                    </property>
                    <property name="text">
                     <string>上一页</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="downButton">
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(41, 41, 49);</string>
                    </property>
                    <property name="text">
                     <string>下一页</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="verticalLayoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>350</y>
                   <width>781</width>
                   <height>171</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QListWidget" name="listWidget_2"/>
                  </item>
                 </layout>
                </widget>
               </widget>
              </widget>
              <widget class="QWidget" name="page_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
